packages/react/src/themes/teams-high-contrast/components/Button/buttonVariables.ts

```diff
@@ -10,25 +10,16 @@ export default (siteVars: any): Partial<ButtonVariables> => {
     backgroundColorFocus: siteVars.accessibleYellow,
     backgroundColorDisabled: siteVars.accessibleGreen,
     borderColor: siteVars.white,
-    borderColorActive: 'transparent',
     borderColorHover: 'transparent',
-    borderColorFocus: siteVars.black,
-    borderColorFocusIndicator: siteVars.white,
 
     primaryColor: siteVars.white,
-    primaryColorActive: siteVars.black,
     primaryColorHover: siteVars.black,
     primaryColorFocus: siteVars.black,
     primaryBackgroundColor: siteVars.black,
     primaryBackgroundColorActive: siteVars.accessibleYellow,
     primaryBackgroundColorHover: siteVars.accessibleYellow,
     primaryBackgroundColorFocus: siteVars.accessibleYellow,
```

Comment:
`primaryBackgroundColorFocus` should be cyan. In general to keep things consistent for hc, elements should get

- Focused background - cyan
- Hover background - yellow
- 	Focus/hover foreground - black
- 	Focus border - 1px cyan border, 1px black border

Reply:
once again, tracked by #1300 as this is not directly related to the PR

Reply:
Once the color scheme is in, we can move the Button styles to use the color scheme, and these things should be resolved. I agree with @Bugaa92 that these are not changes in scope of this PR.